src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/netstandard1.0/Microsoft.NET.Sdk.Functions.targets

```diff
@@ -72,7 +72,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
  -->
   <Target Name="_InitializeFunctionsSdk">
     <Error Text="The ExtensionsMetadataGenerator package was not imported correctly. Are you missing '$(_ExtensionsMetadataGeneratorTargetsPath)' or '$(_ExtensionsMetadataGeneratorPropsPath)'?"
-           Condition="'$(_ExtensionsMetadataGeneratorTargetsImported)' == '' Or '$(_ExtensionsMetadataGeneratorPropsImported)' == ''" />
+           Condition="($(AzureFunctionsVersion) == 'v2' OR $(AzureFunctionsVersion) == 'v2-prerelease' )) And ('$(_ExtensionsMetadataGeneratorTargetsImported)' == '' Or '$(_ExtensionsMetadataGeneratorPropsImported)' == '')" />
```

Comment:
@fabiocav / @soninaren -- is this the best way to know that ExtensionsMetadataGenerator is required to be imported for a project? Wasn't sure if there was a better check I could do.

Reply:
I think this is a reasonable approach.

Reply:
Yup, this is good, for a v1 project the property would be either set to v1 or would not be present.